apps/site/src/lib/components/forms/shared/BaseForm.tsx

```diff
@@ -36,12 +42,25 @@ export default function BaseForm({ children }: PropsWithChildren) {
 		// Use other values when selected
 		for (const field of FIELDS_WITH_OTHER) {
 			const otherField = `other_${field}`;
-			if (formData.get(field) === "other") {
-				formData.set(field, formData.get(otherField)!);
-				formData.delete(otherField);
-			}
+			const otherFieldValue = formData.get(otherField);
+
+			formData.delete(otherField);
```

Comment:
Issue: one of the mentor form fields is currently named `other_questions` and becomes removed here, so the field would need to be renamed.

Reply:
I don't see the field being deleted on my end when I print all the form data out. The loop only takes fields that are in the `FIELDS_WITH_OTHER` array and "questions" is not one of them.

![image](https://github.com/user-attachments/assets/0b017490-6240-4f90-a52b-1b35d43cd1cb)

Reply:
Oh my bad, seems I somehow couldn't find it in the output. Perhaps still a good idea to rename it to reduce chance of collision, or better yet, use a different prefix for the actual other fields, e.g. `_other_pronouns`.